bin/upload-dist.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
This uploads all the individual packages to S3 during CI. Online installs download at runtime and airgap bundles download these when the Job runs.